mypy/subtypes.py

```diff
@@ -143,6 +143,8 @@ def visit_tuple_type(self, left: TupleType) -> bool:
                 return all(is_subtype(item, target_item_type)
                            for item in left.items)
             elif (is_named_instance(right, 'typing.Iterable') or
+                  is_named_instance(right, 'typing.Sized') or
```

Comment:
This isn't right as `Sized` isn't generic, unlike the other types, so `right.args[0]` will fail below. May be worth adding a test case for this.